test/subset-band-name/SubsetBandName_Regression.ipynb

```diff
@@ -0,0 +1,242 @@
+    "- SIT: https://harmony.sit.earthdata.nasa.gov\n",
+    "- Local: http://localhost:3000"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68b5e491-086c-48c0-b9bd-42ae068aa8f4",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "harmony_host_url = 'https://harmony.uat.earthdata.nasa.gov'\n",
+    "# harmony_host_url = 'https://harmony.sit.earthdata.nasa.gov'\n",
+    "# harmony_host_url = 'http://localhost:3000'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37cfa0f5-aa39-453b-b69e-1398472154b7",
+   "metadata": {},
+   "source": [
+    "## Identify Harmony Environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9e8b905a-b08c-4288-80b2-42af52ed0241",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "host_environment = {\n",
+    "    'http://localhost:3000': Environment.LOCAL,\n",
+    "    'https://harmony.sit.earthdata.nasa.gov': Environment.SIT,\n",
+    "    'https://harmony.uat.earthdata.nasa.gov': Environment.UAT,\n",
+    "    'https://harmony.earthdata.nasa.gov': Environment.PROD,\n",
+    "}\n",
+    "\n",
+    "\n",
+    "harmony_environment = host_environment.get(harmony_host_url)\n",
+    "\n",
+    "if harmony_environment is not None:\n",
+    "    harmony_client = Client(env=harmony_environment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ce4a5f7-7f82-455c-b693-52715eed525d",
+   "metadata": {},
+   "source": [
+    "## Setting up Collection Environment Variables\n",
+    "\n",
+    "The cell below sets up the Collection, Granule and other necessary variables for each tested dataset. The datasets provided are in the `UAT` environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da8fbd43-8232-496b-a57a-6156447a965c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mod02hkm_non_production_info = {\n",
+    "    'collection': Collection(id='C1260442414-LAADSCDUAT'),\n",
+    "    'granule_id': 'G1261680941-LAADSCDUAT',\n",
+    "    'variable': ['Band_250M'],\n",
+    "}\n",
+    "\n",
+    "mod02hkm_production_info = {\n",
+    "    'collection': Collection(id='C1378577630-LAADS'),\n",
+    "    'granule_id': 'G2796405746-LAADS',\n",
+    "    'variable': ['Band_250M'],\n",
+    "}\n",
+    "\n",
+    "file_indicators = {'MOD02HKM': 'MOD02HKM.A2023309.2305.061.2023310073403_output.hdf'}\n",
+    "\n",
+    "reference_data = {\n",
+    "    'MOD02HKM': 'reference_data/MOD02HKM.A2023309.2305.061.2023310073403_ref.hdf'\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "722b554f-d69d-45bf-acfe-2ad255f875c0",
+   "metadata": {},
+   "source": [
+    "These selected collections and granules are only available in UAT environment. To minimize the output, all requests will utilize variable subsetting."
```

Comment:
Looks like it was a copy and paste from geoloco which uses variable subsetting to speed up processing time by not reprojecting all datasets.

Reply:
Okay, that makes sense for where it came from then. I think the question still stands, though: saying we're doing variable subsetting to minimise output size seems a bit misleading. We're doing variable subsetting because that's what the service does.